workflow_avamb/README.md

```diff
@@ -147,7 +147,7 @@ Using a GPU can speed up Avamb considerably - especially when you are binning mi
    "minimap_mem": "15GB",
    "minimap_ppn": "10",
    "avamb_mem": "10GB",
-   "avamb_ppn": "10",
+   "avamb_ppn": "10:gpus=1",
```

Comment:
This was deleted in one of the last few commits - it should still be here, right?